yamllint/cli.py

```diff
@@ -192,12 +204,8 @@ def run(argv=None):
             conf = YamlLintConfig(content=args.config_data)
         elif args.config_file is not None:
             conf = YamlLintConfig(file=args.config_file)
-        elif os.path.isfile('.yamllint'):
-            conf = YamlLintConfig(file='.yamllint')
-        elif os.path.isfile('.yamllint.yaml'):
-            conf = YamlLintConfig(file='.yamllint.yaml')
-        elif os.path.isfile('.yamllint.yml'):
-            conf = YamlLintConfig(file='.yamllint.yml')
+        elif find_project_config():
+            conf = YamlLintConfig(file=find_project_config())
```

Comment:
Ideally it would be best to avoid calling `find_project_config()` twice (but also to avoid calling it before the `if/elif` block if not needed). Maybe with a walrus operator `:=`, added in Python 3.8? (but yamllint still supports 3.7...) Or another way?

Reply:
Let me call the function before going through all the loops, and only load the file in that spot. I noticed that the CI runs against Python 3.7 as well, so did not go for the `:=` which would have been convenient.